frontend/public/components/storage/create-pvc.tsx

```diff
@@ -269,6 +269,7 @@ export const CreatePVCForm: React.FC<CreatePVCFormProps> = (props) => {
             key={radio.value}
             onChange={handleVolumeMode}
             inline
+            disabled={storageProvisioner.includes('cephfs') && radio.value === 'Block'}
```

Comment:
If it's RBD then we have to disable the 'FileSystem' too if I am not mistaken.

Reply:
I think rbd supports both types, @vbnrh can you check with QE.